langlib/lang.runtime/src/main/ballerina/runtime.bal

```diff
@@ -36,3 +49,17 @@ public function registerListener(DynamicListener 'listener) = @java:Method {
 public function deregisterListener(DynamicListener 'listener) = @java:Method {
     'class: "org.ballerinalang.langlib.runtime.Registry"
 } external;
+
+# Halts the current strand for a predefined amount of time.
+#
+# + seconds - An amount of time to sleep in seconds
+public isolated function sleep(decimal seconds) = @java:Method {
+    'class: "org.ballerinalang.langlib.runtime.Sleep"
+} external;
+
+# Retrieves the array of `CallStackElement`.
+#
+# + return - An array of `CallStackElement`
+public isolated function getCallStack() returns CallStackElement[] = @java:Method {
```

Comment:
Spec define getStackTrace() function. https://github.com/ballerina-platform/ballerina-spec/blob/master/lang/lib/runtime.bal#L50.

Reply:
Fixed in c6d4744